crates/bevy_math/src/primitives/dim2.rs

```diff
@@ -108,6 +128,24 @@ pub struct Polyline2d<const N: usize> {
 }
 impl<const N: usize> Primitive2d for Polyline2d<N> {}
 
+impl<const N: usize> FromIterator<Vec2> for Polyline2d<N> {
+    fn from_iter<I: IntoIterator<Item = Vec2>>(iter: I) -> Self {
+        let mut vertices: [Vec2; N] = [Vec2::ZERO; N];
+
+        for (index, i) in iter.into_iter().take(N).enumerate() {
+            vertices[index] = i;
+        }
+        Self { vertices }
+    }
+}
+
+impl<const N: usize> Polyline2d<N> {
+    /// Create a new `Polyline2d` from an array of vertices
```

Comment:
It's not necessarily an array anymore. Same thing for other constructors with an `IntoIterator` input

Reply:
So should I change it to "an iterable list of vertices"?

Reply:
I'm not sure what the ideal concise wording for types that implement `IntoIterator` would be... iterable works, but you could also avoid it altogether with something like this:
```suggestion
    /// Create a new `Polyline2d` from its vertices
```

Reply:
I applied this to the rest of the constructors that had this issue 👍